devdocs/design.md

```diff
@@ -2,7 +2,7 @@
 
 ##Architecture
 
-![High Level Architecture](https://raw.githubusercontent.com/TEAMMATES/repo/master/src/main/webapp/dev/images/highlevelArchitecture.png)
+![High Level Architecture](https://raw.githubusercontent.com/TEAMMATES/repo/master/devdocs/images/highlevelArchitecture.png)
```

Comment:
Relative paths work on `process.md`, it should work here as well. In particular, the images should be correctly referenced in [this file](https://github.com/mikelimantara/repo/blob/4802-remove-src-main-webapp-dev-folder/devdocs/design.md).